### PR TITLE
Improve auto-rendered product pages

### LINKED ISSUE #188: Improve auto-rendered product pages
Every auto-rendered product page element should be editable via the Design editor.

Check the auto-rendered product page image target width.

Fix auto-rendered product page meta tags

### PR DESCRIPTION
**Pull request by PeeterPaal**
Closes #188

### CODE REVIEW

File: components/template-head.tpl [hunk 1]
```diff
@@ -17,7 +17,7 @@
 {% if editmode %}
   <link rel="stylesheet" href="{{ site.static_asset_host }}/libs/edicy-tools/latest/edicy-tools.css">
 {% endif %}
-{% stylesheet_link "main.min.css?v=longyearbyen-1.2.1" %}
+<link href="{{ stylesheets_path }}/main.min.css?v={{ template_settings.version }}" media="screen" rel="stylesheet" type="text/css"/>
 {% comment %}{% stylesheet_link "main.css" %}{% endcomment %}
```
Comment (tanelj) [inline components/template-head.tpl:21]: Should be:
```suggestion
<link href="{{ stylesheets_path }}/main.css?v={{ template_settings.version }}" media="screen" rel="stylesheet" type="text/css"/>
```

**Review by tanelj — CHANGES_REQUESTED** [review]
Nice but one minor issue.

**Review by tanelj — APPROVED** [review]
Nice!

**Review by pxska — APPROVED** [review]
Looks good!

**Pull request merged**